src/core/load-fragment.c

```diff
@@ -1364,12 +1365,22 @@ int config_parse_numa_mask(const char *unit,
                            void *userdata) {
         int r;
         NUMAPolicy *p = data;
+        _cleanup_free_ char *all = NULL;
 
         assert(filename);
         assert(lvalue);
         assert(rvalue);
         assert(data);
 
+        if (streq(rvalue, "all")) {
+                r = numa_mask_all_string(&all);
+                if (r < 0) {
+                        log_error_errno(r, "Failed to determine maximum NUMA node index, falling back to 1024: %m");
+                        rvalue = "0-1024";
```

Comment:
that's 1025 cpus, no? that's weird, no?

Reply:
I guess the number doesn't really matter here. IIRC maximum number of NUMA nodes in kernel compile time option so  1024 is as good as any but I agree that total of 1025 is weird.